spec/inputsOutputs/inputsOutputs.spec.js

```diff
@@ -133,23 +142,23 @@ describe('inputsOutputs page', function() {
       components.layout.goToInputsOutputs();
 
       components.inputsOutputs.page.deleteInputOrOutput(inputElement, 'input');// try to delete input
-      expect(components.layout.isElementDisplayed(element(by.css('.popover')))).toBe(true);//popover should show up
+      expect((element(by.css('.popover')).isDisplayed())).toBeTruthy();//popover should show up
 
       components.popovers.popover.clickNo();//click no btn
-      expect(components.layout.isElementDisplayed(element(by.css('.popover')))).toBe(false);//popover should hide
+      browser.sleep(200);
+      expect(element(by.css('.popover')).isPresent()).toBe(false);
```

Comment:
decide on one way to check for popover:
+      expect((element(by.css('.popover')).isDisplayed())).toBeTruthy();
+      expect(element(by.css('.popover')).isPresent()).toBe(false);